sources/packages/web/src/views/institution/locations/active-applications/LocationActiveApplicationSummary.vue

```diff
@@ -58,14 +59,19 @@ export default {
       type: Number,
       required: true,
     },
-    locationName: {
-      type: String,
-      required: true,
-    },
   },
-  setup() {
+
+  setup(props: any) {
+    const store = useStore();
     const tab = ref("active-application-tab");
-    return { tab, ActiveApplicationTab };
+
+    const locationName = computed(() => {
+      return store.getters["institution/myInstitutionLocations"].find(
```

Comment:
This logic can be moved to a composable. Please refer to useStudentStore.ts as example.